Tribler/community/demers/__init__.py

```diff
@@ -0,0 +1,3 @@
+'''
+The demers package contains TTTODO: what does this do?
```

Comment:
The demers package contains a Dispersy community which can be used to verify the speed of message dissemination